README.markdown

```diff
@@ -7206,6 +7206,7 @@ If a number-like argument is specified (including strings that look like numbers
 If a non-number-like string argument is specified, then it is interpreted as a "pattern". The following patterns are supported:
 
 * `'*a'`: reads from the socket until the connection is closed. No end-of-line translation is performed;
+* `'*b'`: the BSD style receive, return anything once socket receives;
```

Comment:
Better be consistent: "return" => "returns".

Also, this is better:

"returns anything the socket has already received (won't return if it has not yet received any data)."